### PR TITLE
Update english and french tutorials

### PR DESCRIPTION
**Pull request by armandsalek**
About the french tutorial, I decided to use french typography for Catala comments and legislative texts. So I replaced : 

- english quotes `"` by  french quotes « and ». 
- quote « ’ » by this one « ’ » 

Thanks to the «[ BÉPO ](https://bepo.fr/wiki/Accueil)» layout, which is an optimized french keyboard.

### CODE REVIEW

File: examples/tutorial_en/tutorial_en.catala_en [hunk 1]
```diff
@@ -592,7 +592,7 @@ scope Test5:
     else $0 # Insert some other modes of computation here
 ```
 
-Here the `NewIncomeTaxComputationFixed of` syntax triggers a call of the scope
+Here the `NewIncomeTaxComputationFixed with` syntax triggers a call of the scope
```
Comment (AltGr) [inline examples/tutorial_en/tutorial_en.catala_en:595]:
```suggestion
Here the `output of NewIncomeTaxComputationFixed with` syntax triggers a call of the scope
```

File: examples/tutorial_en/tutorial_en.catala_en [hunk 2]
```diff
@@ -953,7 +953,7 @@ declaration round_up_100
     content money
     depends on money_amount content money
   equals
-    (money_amount / 100.0 + $0.49) * 100.0
+    round of (money_amount / 100.0 + $0.49) * 100.0
```
Comment (AltGr) [inline examples/tutorial_en/tutorial_en.catala_en:956]: Good catch!

File: examples/tutoriel_fr/tutoriel_fr.catala_fr [hunk 3]
```diff
@@ -1,122 +1,153 @@
...
-s'appelle programmation littéraire, un paradigme de programmation inventé par le
-célèbre informaticien Donald Knuth dans les années 70.
+s’appelle la programmation littéraire, un paradigme de programmation inventé
+par le célèbre informaticien Donald Knuth dans les années 70.
 
-## Définir un impôt sur le revenu fictif
+## Définir un impôt fictif sur le revenu
 
-Le contenu de l'article 1 utilise beaucoup d'éléments du contexte implicite :
+Le contenu de l’article 1 utilise beaucoup d’éléments contextuelles implicites :
```
Comment (AltGr) [inline examples/tutoriel_fr/tutoriel_fr.catala_fr:85]:
```suggestion
Le contenu de l’article 1 utilise beaucoup d’éléments contextuels implicites :
```

File: examples/tutoriel_fr/tutoriel_fr.catala_fr [hunk 4]
```diff
@@ -125,12 +156,28 @@ déclaration champ d'application CalculImpôtRevenu:
   résultat impôt_revenu contenu argent
 ```
 
+Le champ d'application est l’unité d’abstraction de base dans des programmes
+Catala, et la déclaration du champ d’application est semblable à une signature
+de fonction : elle contient une liste de tous les paramètres ainsi que leurs
+types. Mais dans Catala, les variables d’un champ d’application représentent
+trois choses: paramètres d’entrées, paramètres locales et paramètres de
```
Comment (AltGr) [inline examples/tutoriel_fr/tutoriel_fr.catala_fr:163]:
```suggestion
trois choses: paramètres d’entrées, paramètres locaux et paramètres de
```

File: examples/tutoriel_fr/tutoriel_fr.catala_fr [hunk 5]
```diff
@@ -125,12 +156,28 @@ déclaration champ d'application CalculImpôtRevenu:
...
+résultats. La différence entre les trois catégories peuvent être spécifiées par
+les différents attributs d’entrée/résultat qui précèdent les noms de variables.
+« entrée » signifie que la variable doit être définie seulement quand le champ
+d’application « CalculImpôtRevenu » est appelé.
+« interne » signifie que la variable ne peut pas voir en dehors du champ
```
Comment (AltGr) [inline examples/tutoriel_fr/tutoriel_fr.catala_fr:168]:
```suggestion
« interne » signifie que la variable ne peut pas être vue en dehors du champ
```

**Comment by denismerigoux** [comment]
Thanks very much!

**Pull request merged**